azure-governance-cloudwise/CloudWiseDeploy.ps1

```diff
@@ -0,0 +1,56 @@
+﻿cls
```

Comment:
remove this file and ensure the sample deploys with the scripts in the root of the repo